README.md

```diff
@@ -31,21 +42,14 @@ In contrast to the plain `aws_iam_role` resource this module simplifies adding I
   Create an inline IAM policy, Attach custom or AWS managed policies. Create an IAM instance profile.
 
 ## Getting Started
-Basic usage for granting an AWS Account with Account ID `123456789012` access to assume a role that grants
-full access to AWS Simple Storage Service (S3)
+
+Basic usage:
```

Comment:
please do not change already written documentation back to the template

Reply:
resolved in d93f36f